rootfs/rootfs/etc/rc.d/prltoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/bin/sh /usr/local/etc/init.d/prltoolsd start
```

Comment:
> Is this file not executable as-is? 

Ah, sorry. It's executable, but this is a script with a `#!/bin/bash` "shebang" in the first line, which is not available in the boot2docker. However, it works fine under `/bin/sh` interpreter. I think, we should fix that in the next release of Parallels Tools.

> if I run this daemon on VMware or VirtualBox, will the daemon stay resident, or will it terminate appropriately?

This daemon will terminate appropriately, so it works only under Parallels virtualization system. As I know, @SvenDowideit has already verified that scenario with VirtualBox. If you want to test it too, please contact me at e-mail and we'll provide you an access to Parallels Desktop 11 pre-release.